Lib/test/test_asyncio/test_proactor_events.py

```diff
@@ -346,14 +346,31 @@ def test_pause_resume_reading(self):
         for i in range(10):
             self.loop._run_once()
         self.protocol.data_received.assert_called_with(b'data2')
+
+        # pause_reading is idepotent
+        tr.pause_reading()
+        tr.pause_reading()
+        self.assertTrue(tr._paused)
+
         tr.resume_reading()
         self.assertFalse(tr._paused)
         self.loop._run_once()
         self.protocol.data_received.assert_called_with(b'data3')
         self.loop._run_once()
         self.protocol.data_received.assert_called_with(b'data4')
+
+        # resume_reading is idepotent
+        tr.resume_reading()
+        tr.resume_reading()
+
         tr.close()
 
+        # pause/resume reading is no-op on closed transport
+        tr.pause_reading()
+        self.assertFalse(tr._paused)
+        tr._paused = True
```

Comment:
We usually prefer methods to properties: see `Transport.is_closing()` for instance. Anyways, do you use `_paused` attribute anywhere in aiohttp? Maybe we should create a separate issue to discuss this.